scripts/migrations/README.md

```
@@ -0,0 +1,165 @@
+python scripts/migrations/drop_holds_detected_column.py --verify-only
+
+# Rollback (re-add empty column)
+python scripts/migrations/drop_holds_detected_column.py --rollback
+```
+
+## Running Migrations
+
+### Prerequisites
+
+1. **Always backup your database first!**
+2. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)